gateway/config.py

```
@@ -42,8 +42,17 @@ class GatewayConfig(BaseSettings):
     # ロギング
     log_level: str = Field("INFO", alias="LOG_LEVEL")
 
+    # 開発用リロード（本番では false 推奨）
```

Ruff (0.15.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)